packs/data/feats.db/vigorous-health.json

```
@@ -11,14 +11,15 @@
             "value": null
         },
         "description": {
-            "value": "<p>Your physique is robust and can withstand blood loss startlingly well. Whenever you would gain the @Compendium[pf2e.conditionitems.Drained]{Drained}condition, you can attempt a DC 17 flat check. On a success, you don't gain the drained condition.</p>"
+            "value": "<p>Your physique is robust and can withstand blood loss startlingly well. Whenever you would gain the @Compendium[pf2e.conditionitems.Drained]{Drained} condition, you can attempt a @Check[type:flat|dc:17]{flat check}. On a success, you don't gain the drained condition.</p>"
```

```suggestion
            "value": "<p>Your physique is robust and can withstand blood loss startlingly well. Whenever you would gain the @Compendium[pf2e.conditionitems.Drained]{Drained} condition, you can attempt a @Check[type:flat|dc:17]. On a success, you don't gain the drained condition.</p>"
```
Should be no need for the bracket, it should just display "DC 17 Flat Check" by default.